lib/plugins/aws/plan/lib/naming.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+function getStackName(plugin) {
+  return plugin.provider.naming.getStackName();
+}
+
+function getChangeSetName(plugin) {
+  return plugin.changeSetName;
+}
+
+function getS3BucketName(plugin) {
+  return plugin.bucketName;
+}
+
+function getChangeSetS3FolderPath(plugin) {
+  return plugin.serverless.service.package.artifactDirectoryName;
+}
+
+function getChangeSetS3FolderUrl(plugin) {
+  const bucketName = getS3BucketName(plugin);
+  const s3Folder = getChangeSetS3FolderPath(plugin);
+  return `https://s3.amazonaws.com/${bucketName}/${s3Folder}`;
```

Comment:
For certain regions, `s3.amazonaws.com` is invalid.

Could use the [`getS3EndpointForRegion()`](https://github.com/serverless/serverless/blob/master/lib/plugins/aws/utils/getS3EndpointForRegion.js) function to replace this.

```javascript
'use strict';
const getS3EndpointForRegion = require('../../utils/getS3EndpointForRegion');


function getChangeSetS3FolderUrl(plugin) {
  const bucketName = getS3BucketName(plugin);
  const s3Folder = getChangeSetS3FolderPath(plugin);
  const s3Endpoint = getS3EndpointForRegion(plugin.provider.getRegion());
  return `https://${s3Endpoint}/${bucketName}/${s3Folder}`;
}

```

Reply:
nice catch!, will fix it ASAP. Thanks!

Reply:
fixed in https://github.com/serverless/serverless/pull/7509/commits/7eac973c4c71f17911c985178c2a5c94f3419fba